pkg/api/steve/supportconfigs/generator_test.go

```diff
@@ -0,0 +1,192 @@
+package supportconfigs
+
+import (
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	configmapfakes "github.com/rancher/rancher/pkg/generated/norman/core/v1/fakes"
+	"github.com/rancher/rancher/pkg/managedcharts/cspadapter"
+	"github.com/stretchr/testify/assert"
+	"helm.sh/helm/v3/pkg/release"
+	authv1 "k8s.io/api/authorization/v1"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apiserver/pkg/authentication/user"
+	"k8s.io/apiserver/pkg/endpoints/request"
+	"k8s.io/client-go/kubernetes/fake"
+	//authv1 "k8s.io/client-go/kubernetes/typed/authorization/v1"
```

Comment:
Nit: if no longer using this import, please remove it entirely rather than commenting it out.

Reply:
good catch! Will remove it.